docs/framework/docker/console.md

```diff
@@ -0,0 +1,139 @@
+--
+title: Running Console applications in Docker
+description: Running Console applications in Docker
+keywords: .NET, container, Console, Applications, Docker
+author: spboyer
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net-framework-4.6
+ms.technology: vs-ide-deployment
+ms.devlang: dotnet
+ms.assetid: 85cca1d5-c9a4-4eb2-93e6-4f878de07fd7
+---
+# Running console applications in Windows containers
+
+Console applications are used for many purposes; from simple querying of a status to long running document image processing tasks. In either case, the ability to start up and scale these applications are met with limitations of hardware acquisitions, startup times or running multiple instances.
+
+Moving your console applications to use Docker and Windows containers allows for starting these applications from clean state, perform the operation and then shutdown cleanly. This topic will show the steps needed to move a console application to a Windows based container and start it using a PowerShell script.
```

Comment:
> Moving your console applications to use Docker and Windows containers allows for starting these applications from clean state, perform the operation and then shutdown cleanly.

This sentence reads a little off to me. Maybe:

Moving your console applications to use Docker and Windows Server Containers allows for starting these applications from a clean state, enabling them to perform the operation and then shutdown cleanly.